staging/cse/windows/kubernetesfunc.ps1

```diff
@@ -144,6 +144,7 @@ function Update-DefenderPreferences {
     Add-MpPreference -ExclusionProcess "C:\k\azurecni\bin\azure-vnet.exe"
     Add-MpPreference -ExclusionProcess "C:\k\azurecni\bin\AzureNetworkContainer.exe"
     Add-MpPreference -ExclusionProcess "C:\k\azurecni\bin\CnsWrapperService.exe"
+    Add-MpPreference -ExclusionProcess "C:\k\azurecns\azure-endpoints.json"

```

Comment:
Does C:\k\azurecns exist?

Reply:
It does exist in our CNIv2 (Swift)
It does not exist in CNIV1.

Reply:
Is this going to create issues if this script runs on CNIV1?

Reply:
I think that we need to test it. If it causes issues, we only can add this with CNIv2.

Reply:
I run `Add-MpPreference -ExclusionProcess "C:\k\azurecns\azure-endpoints.json"` in a WS2019 node and it has no issue when `C:\k\azurecns\` does not exist